airbyte-integrations/connectors/source-iterable/source_iterable/api.py

```diff
@@ -10,7 +10,12 @@
 
 import requests
 from airbyte_cdk.models import SyncMode
-from source_iterable.iterable_streams import IterableExportStreamAdjustableRange, IterableExportStreamRanged, IterableStream
+from source_iterable.iterable_streams import (
```

Comment:
seconding Augustin's earlier comment, can we combine `api.py` and `iterable_streams.py` into one file called `streams.py` that will the define all the streams in one place. That is more of the convention that other connectors have adopted and I don't really see what benefit we get from having everything spread across two files.